crates/revm/src/evm_impl.rs

```diff
@@ -275,34 +275,36 @@ impl<'a, GSPEC: Spec, DB: Database, const INSPECT: bool> EVMImpl<'a, GSPEC, DB,
         (new_state, logs, gas_used, gas_refunded)
     }
 
-    /// EVM create opcode for both initial crate and CREATE and CREATE2 opcodes.
-    fn create_inner(
+    fn prepare_create(
         &mut self,
         inputs: &CreateInputs,
-    ) -> (InstructionResult, Option<B160>, Gas, Bytes) {
+    ) -> Result<
+        (Gas, B160, JournalCheckpoint, Box<Contract>),
+        (InstructionResult, Option<B160>, Gas, Bytes),
```

Comment:
This looks ugly, but I dont have any better way atm.

Jus define `pub type .. = ..` at the beginning of the file to satisfy clippy.